pii_recognition/experiments/pii_validation/google_cloud_reports/scores_ner.json

```diff
@@ -0,0 +1,11 @@
+{
+  "exact_match_f1": 0.3466,
+  "partial_match_f1_threshold_at_50%": 0.3527,
+  "frozenset({'EMAIL', 'CREDIT_CARD', 'OTHER'})": 0.066,
```

Comment:
should `CREDIT_CARD` go with `NUMBER` instead of `OTHER`?

Reply:
I think that's a better choice. While the best choice would be grouping `CREDIT_CARD`, `OTHER`, `NUMBER` all together, but this is a non-exclusive grouping strategy in which `OTHER` would belong to multiple groups and this is not supported by our implementation.

Why I choose to put CREDIT_CARD into OTHER is mainly because I saw this example:

Are there any charges applied to widraw money from ATM with the card 5117398483315666

- "card 5117398483315666" -- OTHER

And many other examples would argue to classify CREDIT_CARD to NUMBER as you observed.